core/os/device/deviceinfo/cc/windows/query.cpp

```diff
@@ -72,68 +36,13 @@ void destroyContext() {
   if (--gContextRefCount > 0) {
     return;
   }
-
-  if (gContext.mWnd != nullptr) {
-    DestroyWindow(gContext.mWnd);
-  }
-  if (gContext.mGlCtx != nullptr) {
-    gContext.wglMakeCurrent(gContext.mHDC, 0);
-    gContext.wglDeleteContext(gContext.mGlCtx);
-  }
-}
-
-void createGlContext() {
-  gContext.wglCreateContext =
-      (pfn_wglCreateContext)core::GetGlesProcAddress("wglCreateContext");
```

Comment:
Removing all calls to this function will likely break stuff, as this function is where we keep a hack to work around broken MESA drivers.